core/templates/dev/head/components/forms/schema_editors/SchemaBasedUnicodeEditorDirective.js

```diff
@@ -115,6 +115,10 @@ oppia.directive('schemaBasedUnicodeEditor', [
             return $sce.trustAsHtml(
               $filter('convertUnicodeWithParamsToHtml')($scope.localValue));
           };
+          $scope.onClickPopupDemo = function() {
+            var popup = document.getElementById('myPopup');
```

Comment:
@kefeh do you know about stuff like ng-click, ng-class, etc.? If not, I recommend reading/watching an Angular tutorial and learning about how to do things like this, before working on this PR. In general, in Angular, we try not to reference the DOM directly.

Some possible starting points:
-  https://docs.angularjs.org/tutorial
- https://www.codeschool.com/courses/shaping-up-with-angularjs
- https://egghead.io/technologies/angularjs